### DISCUSSION TITLE
Recent pull request copy / paste behaviour change

### DISCUSSION THREAD

**Discussion by john-n-smith** [community/community#22300] [New to GitHub]
<p>When copying from a pull request the pasteboard will contain formatting characters from the underlying markdown. For instance, copying an item from a list will be prefixed with an asterisk. This is a recent change with the previous behaviour copying only selected characters to the pasteboard.</p>
<p>This behaviour is unintuitive and potentially dangerous. For instance, a PR may contain a markdown list of commands to execute for QA. If copying an item from the list into a prompt, the command will be prefixed with an asterisk, silently modifying the command without alerting the user.</p>
<p>Is it possible to turn this behaviour off?</p>
<p>If the underlying markdown formatting is required, the user can use the ‘edit’ view of the PR.</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/john-n-smith">@john-n-smith</a>,</p>
<p>I’m not certain that this is an intentional change. I need to do a bit more investigation. Could you please tell me what browser you’re using? Also, when you copy text from a pull request, are you copying text from files included in the pull request or from the body of the message?</p>
<p>Thanks! </p>

**Comment by john-n-smith** [comment]
<p>Hi <a class="mention" href="/that-pat">@that-pat</a>, thanks for getting back to me,</p>
<p>I’ve observed this behaviour on both Safari and Chrome, both on OS X. I’m copying from the body of the PR.</p>
<p>Here’s an example where list markdown is being added to the pasteboard although not selected:</p>
<p><img src="https://user-images.githubusercontent.com/1314694/181070110-1d841bbb-ab50-4b02-9527-a46f6c86d45a.gif" alt="PR copy - paste.gif" data-base62-sha1="yDXJWGFba8EVS1vKL13WBss24op" width="350" height="188"></p>
<p>Thanks for you time,</p>
<p>John.</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/john-n-smith">@john-n-smith</a>,</p>
<p>We made a recent change to how this is being handled. Would you mind trying this again and letting me know if it works the way that you expect it to now?</p>
<p>Thanks! </p>

**Comment by john-n-smith** [comment]
<p>That’s now working as expected, thanks for you help!</p>
